<launch>

  <!-- BEGIN ROBOT 1-->
  <group ns="drone1">
    <param name="tf_prefix" value="drone1_tf" />
    <include file="$(find espdrone_driver)/launch/one_drone.launch" >
      <arg name="x" value="0.0" />
	    <arg name="y" value="0.0" />
	    <arg name="z" value="0.0" />
	    <arg name="Y" value="1.57" />
      <arg name="name"  value="drone1" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="drone2">
    <param name="tf_prefix" value="drone2_tf" />
    <include file="$(find espdrone_driver)/launch/one_drone.launch" >
      <arg name="x" value="0.0" />
	    <arg name="y" value="-1.0" />
	    <arg name="z" value="0.0" />
	    <arg name="Y" value="1.57" />
      <arg name="name"  value="drone2" />
    </include>
  </group>

  <!-- BEGIN ROBOT 3-->
  <!--group ns="drone3">
    <param name="tf_prefix" value="drone3_tf" />
    <include file="$(find espdrone_driver)/launch/one_drone.launch" >
      <arg name="x" value="1.0" />
	    <arg name="y" value="-1.0" />
	    <arg name="z" value="0.0" />
	    <arg name="Y" value="1.57" />
      <arg name="name"  value="drone3" />
    </include>
  </group-->
  
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find espdrone_driver)/config/espdrone.rviz" required="true" />
  <node name="tf_publisher" pkg="espdrone_driver" type="tf_broadcaster_imu.py" output="screen"/>
</launch>
